Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -993,6 +990,8 @@ RefPtr<CSSStyleValue const> Parser::parse_counter_definitions_value(TokenStream<
             auto& name_token = function_tokens.consume_a_token();
             if (!name_token.is(Token::Type::Ident))
                 break;
+            if (name_token.token().ident().equals_ignoring_ascii_case("none"sv))
+                break; // "none" is invalid as the name of a <reversed-counter-name>.
```

Comment:
`reversed()` parsing should probably use the same `parse_custom_ident_value()` code as above.